docs/source/fundamentals/setup.rst

```diff
@@ -14,11 +20,8 @@ To install LineaPy, run:
 
     pip install lineapy
 
-Or, if you want the latest version of LineaPy directly from the source, run:
-
-.. code:: bash
-
-    pip install git+https://github.com/LineaLabs/lineapy.git --upgrade
+If you want to run the latest version of LineaPy directly from the source, follow instructions
+:ref:`here <contribution_install>`.
```

Comment:
This reference exists in the dev branch for contributor guide revamp (`LIN-680-produce-first-revamp-of-contributor-guide`), which will be merged to `main` soon.